src/aggregator/client/writer_mgr.go

```diff
@@ -105,7 +105,7 @@ type writerManager struct {
 	closed  bool
 	metrics writerManagerMetrics
 	_       cpu.CacheLinePad
-	pool    xsync.PooledWorkerPool
+	pool    xsync.StaticPooledWorkerPool
```

Comment:
i thought it was better to be explicit with the type named `StaticPooledWorkerPool`. Don't feel strongly about this and we can keep `PooledWorkerPool` to mean static inherently.